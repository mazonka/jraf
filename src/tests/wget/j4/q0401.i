command=jw md 0 /.jraf.sys/a : put 0 /.jraf.sys/b 0 1 Kg== : read 0 /.jraf.sys/a : read 0 /.jraf.sys/b