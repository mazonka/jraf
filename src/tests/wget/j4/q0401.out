OK /.jraf.sys/a : OK 1 : OK 0 -1 0 : OK 0 1 Kg==